commit 1f3e0f33150d130770c374a1f16c8f465e2ac0d7
Author: John Levon <john.levon@joyent.com>
Date:   2019-08-30T14:50:35+00:00 (5 weeks ago)
    
    OS-7973 fbt entry probes incorrectly placed due to -fshrink-wrap
    Reviewed by: Jerry Jelinek <jerry.jelinek@joyent.com>
    Reviewed by: Robert Mustacchi <rm@joyent.com>
    Approved by: Robert Mustacchi <rm@joyent.com>

diff --git a/Makefile b/Makefile
index ee2192b..e478192 100644
--- a/Makefile
+++ b/Makefile
@@ -1,5 +1,5 @@
 #
-# Copyright (c) 2019, Joyent, Inc.
+# Copyright 2019 Joyent, Inc.
 #
 
 include		$(PWD)/../../../build.env
@@ -108,6 +108,7 @@ KERNEL_CFLAGS = \
 	$(ALWAYS_CFLAGS) \
 	-m64 \
 	-mcmodel=kernel \
+	-fno-shrink-wrap \
 	-g \
 	-O \
 	-fno-inline \
